### ISSUE TITLE
Respect `--isolated` option when running with `flake8`

### ISSUE THREAD

**Issue by sobolevn** [terrencepreilly/darglint#55]
Flake8 has a very useful flag: `--isolated` that does not read any config values.

Currently `darglint` still relies on its config value.

How to illustrate this problem?

```py
# ex.py
def some(arg):
    """
    Test.

    Returns some value.
    """
    return 'some'
```

And config:

```ini
# setup.cfg
[darglint]
strictness = long
```

Let's run it:

```
» flake8 ex.py --select=DAR
```

Done! No problems.

Let's run it with `--isolated`:

```
» flake8 ex.py --select=DAR --isolated
```

Done? Why? The default value of `darglint` should not allow this code.

Let's try to remove the config value from `setup.cfg`:

```
» flake8 ex.py --select=DAR           

with_lb.py

  2:1      DAR101 Missing parameter(s) in Docstring: - arg
  """
  ^

  2:1      DAR201 Missing "Returns" in Docstring: - return
  """
  ^
```

Boom! I expect the same to happen with `--isolated` mode.

**Comment by sobolevn** [comment]
You can now place config into `[flake8]` section. And `--isolated` will be respected.
